compute_tools/src/monitor.rs

```diff
@@ -105,10 +105,10 @@ fn watch_compute_activity(compute: &ComputeNode) {
 }
 
 /// Launch a separate compute monitor thread and return its `JoinHandle`.
-pub fn launch_monitor(state: &Arc<ComputeNode>) -> Result<thread::JoinHandle<()>> {
+pub fn launch_monitor(state: &Arc<ComputeNode>) -> Result<thread::JoinHandle<()>, std::io::Error> {
```

Comment:
Is this change relevant?

Reply:
no, removing it

Reply:
https://github.com/neondatabase/neon/pull/4779
moved this change to